customer_satisfaction/tests/data_test.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+import pytest
+from model.data_cleaning import DataCleaning
+from steps.ingest_data import IngestData
+from zenml.steps import step
+
+
+@step
+def data_test_prep_step():
+    """Test the shape of the data after the data cleaning step."""
+    try:
+        ingest_data = IngestData()
+        df = ingest_data.get_data()
+        data_cleaning = DataCleaning(df)
+        df = data_cleaning.preprocess_data()
+        X_train, X_test, y_train, y_test = data_cleaning.divide_data(df)
+
+        assert X_train.shape == (92487, 12), "The shape of the training set is not correct."
```

Comment:
doing it with `assert` statements here makes a lot of sense to me. Why make it more complicated than it needs to be!

Reply:
Thanks